lib/ui/settings.dart

```diff
@@ -78,17 +85,34 @@ class _SettingsScreenState extends State<SettingsScreen> {
                       'name'.tr,
                       style: kSettingsTextW700
                     ),
-                    SizedBox(
-                      height: 36,
-                      width: 100,
-                      child: TextField(
-                        style: kSettingsTextW400,
-                        textAlign: TextAlign.right,
-                        controller: myController,
-                        decoration: InputDecoration(
-                          border: InputBorder.none,
-                          hintText: gc.myPlayer.name,
-                          hintStyle: kSettingsTextW400
+                    Expanded(
+                      child: Focus(
+                        onFocusChange: (focus){
+                          hasFocus = focus;
+                          if(hasFocus==true){
+                            myController.clear();
+                          } else {
+                            myController.text=gc.myPlayer.name;
+                          }},
+                        child: TextField(
+                          style: kSettingsTextW400,
+                          textAlign: TextAlign.right,
+                          textDirection: TextDirection.rtl,
```

Comment:
Does it make sense to set`TextDirection.rtl`?
When I add a space the cursor jumps to the left.

Reply:
I think i added it trying to solve the trailing space that don't work but it should be removed

found the issue on Flutter github https://github.com/flutter/flutter/issues/40648